library/spit_results.py

```diff
@@ -356,7 +356,7 @@ def handle_redhat_packages(self, data):
                         max_build_time = pkg.build_time
                         last_built = pkg
 
-                is_red_hat = "Y" if rh_packages > 0 else "N"
+                is_red_hat = "Y" if not rh_packages else "N"
```

Comment:
Change came from this error that was occurring in Travis build:
https://travis-ci.org/quipucords/rho/jobs/259810259#L701

After looking into it lists and sets that are empty are False. The suggested change was to the above as depending on len() can have negative performance and a list/set that contains values will result in a truthy value.